nuage_metroae_config/actions.py

```diff
@@ -1061,11 +1067,18 @@ def execute(self, writer, context=None):
 
             file_mode = "w" if self.append_to_file is False else "a"
             console_text = ""
+            isZip = False
```

Comment:
Please remove the zip functionality.  This code is supposed to be entirely generic (think about supporting other devices like SROS).  The unzipping is specific to this one use case.  I am ok with the base64 because it is explicitly specified in the template.

Reply:
Okay then the zip file creation will fail and incorrect format zip file is created. The zipping is only done if a .zip extension  is found in the output file. Not all the time.